ci-operator/step-registry/rosa/cluster/wait-ready/operators/rosa-cluster-wait-ready-operators-commands.sh

```diff
@@ -37,8 +60,8 @@ if [[ "${costatus}" =~ "timed out" ]]; then
     curl -X POST -H 'Content-type: application/json' --data "${slack_message}" "${slack_hook_url}"
     sleep 10800
   fi
-
   exit 1
 else
-  echo "All cluster operators are done progressing."
+  record_cluster "timers" "co_wait_time" $(( "${end_time}" - "${start_time}" ))
```

Comment:
if we put the record on the line 50, we dont know if the oc wait clusteroperators has finished because all operators are ok or because of the timeout

setting on the line 65, we ensure that we are saving only valid information when there is no timeout on the log

Reply:
@yasun1 can you confirm this is ok?

Reply:
lgtm